docker/actinia-stac-tests/start.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# TODO: read paths from config
+mkdir -p /actinia_core/grassdb
+mkdir -p /actinia_core/userdata
+mkdir -p /actinia_core/workspace/temp_db
+mkdir -p /actinia_core/workspace/tmp
+mkdir -p /actinia_core/resources
+
+# copy pgpass from mounted (!) file
+cp /mnt/pgpass/.pgpass $HOME/.pgpass
+chmod 0600 $HOME/.pgpass
+
+# copy db.login file from mounted (!) file
+cp /mnt/pgpass/.grass7 $HOME/.grass7/dblogin
+chmod 0600 $HOME/.grass7/dblogin
+
+# copy db.login file to actinia-core tmp location
+mkdir -p /tmp/:/root/.grass7
+cp /root/.grass7/dblogin /tmp/:/root/.grass7/
```

Comment:
As no docker-compose.yml is present here and no mount is used anywhere, these lines can be removed.